### ISSUE TITLE
Could not find a device with a UDID or name matching...

### ISSUE THREAD

**Issue by zwiebelspaetzle** [calabash/run_loop#615]
When running `cucumber` on my ios app, I get the error 
`Could not find a device with a UDID or name matching 'iPhone 7 (10.2)' (ArgumentError)
 ./features/support/01_launch.rb:27:in 'Before'`.  

I'm running my project in the device Simulator, not a physical device.

I set up calabash with calabash-sandbox, per  https://developer.xamarin.com/guides/testcloud/calabash/quickstarts/xcode/.  It used to work, but now (perhaps caused by Xcode update) I get that error.  I've tried searching through my codebase for `iPhone 7 (10.2)` and `DEVICE_TARGET` but can't find where that is specified.  I've also tried running from Xcode with iOS 10.3 as the iOS Deployment Target, but I get the same error.

How can I get calabash to stop looking for that specific device?

Thanks!

**Comment by jmoody** [comment]
> 'iPhone 7 (10.2)'

This is the default simulator for Xcode 8.2.*.  This simulator is not automatically available with Xcode 8.3, it must be downloaded and installed (from Xcode > Settings > Components).  My best guess is that you are using Xcode 8.3 and old version of run-loop.   Update your run-loop version to at least 2.4.0.

> How can I get calabash to stop looking for that specific device?

```
# Look for the UDID of the simulator you want
$ xcrun simctl list devices

$ DEVICE_TARGET=<UDID> bundle exec cucumber
```

**Comment by zwiebelspaetzle** [comment]
That did the trick.  Thanks!
